src/main/java/org/documentoviscode/splashyapi/domain/PartnershipAgreement.java

```diff
@@ -0,0 +1,23 @@
+package org.documentoviscode.splashyapi.domain;
+
+import jakarta.persistence.*;
+
+import java.time.LocalDate;
+
+@Entity
+@Table(name="Partnership_Agreements")
+public class PartnershipAgreement extends Document{
```

Comment:
Umowa biznesowa to raczej `PartnershipContract`, agreement znaczy bardziej dogadanie się